test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_config_test.clj

```diff
@@ -93,8 +103,37 @@
 
     (is (expected-https-config?
           (merge valid-ssl-pem-config
-                 {:ssl-host "foo.local" :ssl-port 8001 :request-header-max-size 16192})
-          {:https {:host "foo.local" :port 8001 :request-header-max-size 16192}})))
+                 {:ssl-host "foo.local"
+                  :ssl-port 8001
+                  :request-header-max-size 16192})
+          {:https {:host "foo.local"
+                   :port 8001
+                   :request-header-max-size 16192}}))
+
+    (is (expected-https-config?
+          (merge valid-ssl-pem-config
+                 {:ssl-host "foo.local"
+                  :ssl-port 8001
+                  :so-linger-seconds 22})
```

Comment:
Ah, well in that case it's a moot point and nevermind.  Sorry, I even read
the BSD man page and it didn't sink in since the JVM layer exposed the
setting as milliseconds.